curve-ansible/README.md

```diff
@@ -374,7 +374,6 @@ ansible-playbook rolling_update_nebd.yml -i client.ini --tags restart
 │   ├── get_curve_version_from_metric.yml                   # 从metric获取curve版本
 │   ├── get_distro_name.yml                                 # 获取系统版本
 │   ├── get_nebd_version_from_metric.yml                    # 从metric获取nebd版本
-│   ├── get_software_version_from_package_version.yml       # 从包版本获取软件版本
```

Comment:
if this file is no longer in use, it can be deleted

Reply:
> if this file is no longer in use, it can be deleted

done